core/templates/dev/head/pages/collection_player/collection_player.html

```diff
@@ -58,6 +71,113 @@
         left: 42px;
         position: absolute;
       }
+      @media only screen and (max-width : 600px)
+      {
+        .oppia-collection-player-title-font
+        {
+          font-size: 32px;
+        }
+      }
+      exploration-summary-tile md-card {
```

Comment:
Ah, I see. But doesn't `exploration-summary-tile md-card` apply to both of these?